back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -66,7 +66,7 @@ def index_idea_markers
       current_user: current_user
     ).find_records
     ideas = paginate SortByParamsService.new.sort_ideas(ideas, params, current_user)
-    ideas = ideas.includes(:author, :topics, :project, :idea_status, :idea_files)
+    ideas = ideas.includes(:author, :topics, :project, :idea_status)
```

Comment:
@kogre, I think we could consider adding this one back. I don't immediately see why it is included for the idea markers but not when indexing ideas, except maybe because the user can browse ideas on the map without leaving the page.

I think the other changes to the includes look fine.